src/pages/management/configuration-api/index.mdx

```diff
@@ -30,6 +30,8 @@ All the LiveChat OAuth 2.1 access tokens have a prefix: `dal:` or `fra:`. This p
 
 Summing up, if the user token starts with `fra`, you should add the `X-Region: fra` header. If the token starts with `dal`, you don’t have to specify the header.
 
+If you get the [`misdirected_request`](#possible-errors) error, use the prefix returned in the error as the value of `X-Region`.
```

Comment:
I added this sentence to all versions of Agent Chat Web API, Customer Chat Web API, and Configuration API.

Not sure about how to resolve this problem in RTM